src/main/java/com/github/devotedmc/hiddenore/Config.java

```diff
@@ -47,23 +51,49 @@ private Config() {
 		ignoreSilktouch = false;
 		defaultPrefix = "You found hidden ore!";
 		transformAttemptMultiplier = 3;
+		configurations = new HashMap<String, FileConfiguration>();
 	}
 
 	public static void loadConfig() {
 		try {
-			file = HiddenOre.getPlugin().getConfig();
-			doLoad();
+			//TODO default-world.yml is never generated, but is required to generate per-world configs.
+			mainConfig = HiddenOre.getPlugin().getConfig();
+			doLoadMainConfig(mainConfig);
+			for (World world : HiddenOre.getPlugin().getServer().getWorlds()) {
+				File saveFile = new File(HiddenOre.getPlugin().getDataFolder(), String.format("%s-config.yml", world.getName()));
+				if (saveFile.exists()) {
+					//TODO pretty sure this whole scope here doesn't work.
+					//TODO double check if this is supposed to be the specific config or default-world.yml for the reader. I'm too tired to be able to follow this around in my head right now.
+					HiddenOre.getPlugin().getLogger().info("Loading configs for world " + world.getName());
+					Reader worldExistsReader = new InputStreamReader(HiddenOre.getPlugin().getResource(String.format("%s-config.yml", world.getName())));
```

Comment:
getResource() pulls from the .jar file -- not the "file system" iirc.

Just use 

```
Reader worldExistsReader = new InputStreamReader(new FileReader(saveFile));
```